src/main/java/com/timeplus/kafkaconnect/TimeplusSinkTask.java

```diff
@@ -133,20 +141,16 @@ private String getCreateRawPayload(String stream) {
         JSONObject payload = new JSONObject();
         payload.put("name", stream);
 
-        JSONObject col = new JSONObject();
-        col.put("name", "raw");
-        col.put("type", "string");
-
         JSONArray cols = new JSONArray();
-        cols.put(col);
+        cols.put(RAW_COL);
         payload.put("columns", cols);
 
         return payload.toString();
     }
 
     private String getCreateJSONPayload(String stream, String event) {
```

Comment:
nit: would it make more sense to name this method `getCreateStreamPayload`?

Reply:
renamed